src/components/posts/comments/comment-element.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+	export let id: string;
+	export let comment: string;
+	import { Avatar } from '@skeletonlabs/skeleton';
+	let shortComment: string = comment.substring(0, 200);
+
+	//function to toggle wether the comment is shown fully or not (only for long comments)
+	function toggleFullComment(): void {
+		let shortCommentElement: HTMLElement = document.getElementById(id + '-short-comment')!;
+		let fullCommentelement: HTMLElement | null = document.getElementById(id + '-full-comment')!;
+		let button: HTMLElement | null = document.getElementById(id + '-comment-button')!;
+		if (shortCommentElement.style.display === 'none') {
+			shortCommentElement.style.display = 'block';
+			fullCommentelement.style.display = 'none';
+			button.innerText = 'Mehr';
+		} else {
+			shortCommentElement.style.display = 'none';
+			fullCommentelement.style.display = 'block';
+			button.innerText = 'Weniger';
+		}
+	}
+</script>
+
+<div class="grid grid-cols-[auto_1fr] gap-2">
+	<Avatar src="src/images/about/jonas.png" width="w-8 md:w-10 min-w-0" />
+	<div
+		class="card p-2 variant-soft-tertiary dark:variant-soft-surface rounded-tl-none space-y-2 min-w-0"
+	>
+		<header class="flex justify-between items-center pr-1">
+			<p class="text-xl md:text-2xl font-bold">Username</p>
+			<small class="text-sm md:text-base">12.12.2023 17:45 Uhr</small>
+		</header>
+		<div>
+			<p class="text-xl md:text-2xl break-words w-full" id={id + '-short-comment'}>
+				{shortComment}
+			</p>
+			<p
+				class="text-xl md:text-2xl break-words w-full"
+				id={id + '-full-comment'}
+				style="display:none;"
+			>
+				{comment}
+			</p>
+			{#if comment.length > shortComment.length}
+				<button
+					class="text-xl md:text-2xl text-gray-400 hover:text-gray-500 focus:text-gray-500"
+					id={id + '-comment-button'}
+					on:click={toggleFullComment}>Mehr</button
+				>
+			{/if}
+		</div>
+	</div>
+</div>
```

Comment:
Ich würde das hier nicht mit plain html machen, sondern mit [svelte assignments](https://learn.svelte.dev/tutorial/reactive-assignments)
Beispielsweise: 
  ``` 
let moreClicked = false 
        function changeMoreClicked() {
		moreClicked = !moreClicked ;
	} 
```

Dann kann man das in den Button hauen und mithilfe von `moreClicked ? 'Weniger' : Mehr` den Button umbenennen und du kannst dann auch die Redundanz mit den Kommentaren verringern mit `moreClicked ? 'comment' : shortComment`

Reply:
Wenn comment kleiner oder gleich als shortComment ist es ja egal welchen String man einsetzt und dann muss nur mit einem svelte `{#if comment <= shortComment}` der Button aus oder eingeblendet werden